project/core/templates/how_it_works.html

```diff
@@ -40,15 +39,18 @@ <h4 class="dark-purple">{% trans "Still not sure about something in CiviWiki?" %
         </div>
     </div>
 </script>
+{% endblock backbone_template %}
 
-<body>
+{% block content %}
     {% include "static_nav.html" %}
     <div id="howitworks"></div>
     {% include "static_footer.html" %}
```

Comment:
Let's move the static navigation and footer to the `base.html`.
```suggestion
    <div id="howitworks"></div>
```